user_input.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner; /* Afin d'utiliser les scanners, on doit ajouter de la librairie, la classe Scanner se trouver dans le pack Utility */
+
+public class user_input {
+    public static void main(String[] args){
+
+        Scanner scanner1 = new Scanner(System.in); 
+        /* On initialise un Scanner nommer "scanner1" */
+
+        System.out.println("What is your name? ");
```

Comment:
```suggestion
        System.out.println("What is your name?");
```
Tu peux enlever l'espace à la fin puisque `println` va ajouter un saut de ligne à la fin du print.